app/views/hyrax/base/_metadata.html.erb

```diff
@@ -1,8 +1,4 @@
-<h2><%= t('.header') %></h2>
-<table class="table table-striped <%= dom_class(presenter) %> attributes" <%= presenter.microdata_type_to_html %>>
-  <thead>
-    <tr><th><%= t('.attribute_name_label') %></th><th><%= t('.attribute_values_label') %></th></tr>
-  </thead>
+<table class="table-borderless <%= dom_class(presenter) %> attributes" <%= presenter.microdata_type_to_html %>>
```

Comment:
why are we dropping the `table` class?  why not `table table-borderless`?  If we had that, couldn't we drop a lot of custom styles?

Reply:
That was a mistake, I'll update that and fix the `borderless` to `table-borderless`, thanks for the catch.